src/unit4/lesson2/README.md

```diff
@@ -0,0 +1,74 @@
+# Unit 4, Lesson 2 - Database and Database Group
+
+Here we go! Welcome to second lesson of Unit 4. Now that you already know how to set up a cluster, let's learn some in depth concepts about how it works. Also, let's create learn how to manage a database in the cluster.
+
+## It's all about "Operations"
+Operations in RavenDB can be classified into two categories:
+
+1) cluster-wide operations: that impact the entire cluster. For example,  creating a new database.
+
+2) internal database operations: that impact a single database. For example, creating a new document.
```

Comment:
But creating an index is a cluster wide operation.
Better to say something like: impact the cluster or a database group. Then give the index example as being created on all the instances.